package.json

```diff
@@ -104,7 +104,7 @@
     "@grafana/schema": "^10.1.2",
     "@grafana/ui": "^10.1.2",
     "deepdash": "^4.5.4",
-    "eslint-plugin-jsdoc": "^40.1.1",
+    "eslint-plugin-jsdoc": "48.0.4",
```

Comment:
PR looks good (with some limitations, that we discussed), but we need to bump the version and add a changelog.MD entry

Reply:
- Updated the help text for the custom query button mentioning the fact that we expose the unit and unitExternalId of the original data in the label
- Bumped the package version
- Added a changelog -> let me know if there are other things to mention there